engine/classes/Elgg/Amd/Config.php

```diff
@@ -218,11 +218,18 @@ public function hasModule($name) {
 	 * @return array
 	 */
 	public function getConfig() {
-		return array(
+		$defaults = array(
 			'baseUrl' => $this->baseUrl,
 			'paths' => $this->paths,
 			'shim' => $this->shim,
 			'deps' => $this->getDependencies(),
+			'waitSeconds' => 20,
 		);
+		
+		$params = array(
+			'defaults' => $defaults
+		);
+		
+		return elgg_trigger_plugin_hook('config', 'amd', $params, $defaults);
```

Comment:
Actually I think we should call the hook at the use point of getConfig() rather than having this class depend on it. @ewinslow thoughts? In general it seems good to push minor dependencies like these outside, or require them to be passed into the function for use.